src/Chapter1.hs

```diff
@@ -204,31 +204,31 @@ So, the output in this example means that 'False' has type 'Bool'.
 > Try to guess first and then compare your expectations with GHCi output
 
 >>> :t True
-<INSERT THE RESULT INSTEAD OF THE TEXT>
+Bool
 >>> :t 'a'
-<INSERT THE RESULT INSTEAD OF THE TEXT>
+Char
 >>> :t 42
-<INSERT THE RESULT INSTEAD OF THE TEXT>
+Num p => p
```

Comment:
We haven't covered this in this Chapter as it is a bit more advanced concept, but in reality it will be 
```suggestion
42 :: Num a => a
```
We will explain what it exactly means in Haskell later, but for now, you can think of it as a `Num`eric object. If you look at it this way, it indeed makes sense, as 42 could represent `Int`, `Natural`, `Integer` and even `Double`! So it is a generalised method to say the type of any numeric 🙂 

Btw, we check this example with some test, that is why it requires the exact output of the GHCI on that. Let us know if we could help with running GHCI. If you installed everything that was in the instructions, you should be able to:

```
$ ghci
```

Reply:
Thanks for the remarks, I directly typed the answers from my head and didn't even run `ghci`, maybe I should have done that :)

Reply:
That's great that you checked yourself first, this is actually a very healthy thing to do, great one 💪🏼